notebooks/official/prediction/get_started_with_vertex_private_endpoints.ipynb

```diff
@@ -742,7 +714,7 @@
       "source": [
         "### Create VPC peering for default network\n",
         "\n",
-        "For simplicity, we setup VPC peering to the default network. You can create a different network for your project.\n",
+        "For simplicity, you setup VPC peering to the default network. You can create a different network for your project.\n",
```

Comment:
don't need "you" here. It's implied by the form of the verb (oh, and change "setup", which is a noun, to "set up", which is a verb). 

- For simplicity, set up VPC peering to the default network.

Reply:
Did the required changes